docs/content/users/quickstart.md

```diff
@@ -509,6 +509,7 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
 
     ```bash
     mkdir my-laravel-site && cd my-laravel-site
+    # If you're using DDEV v1.24.8+, you don't need --disable-settings-management=true
     ddev config --project-type=laravel --docroot=public --omit-containers=db --disable-settings-management=true
```

Comment:
```suggestion
    # If you're using  an older version of DDEV (<=v1.24.7), you need to add `--disable-settings-management=true`
ddev config --project-type=laravel --docroot=public --omit-containers=db
```

Reply:
I thought about changing the default later, but I agree, let's do it now.